CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# v1.14.4
+
+## Release v1.14.4 - 2022/01/06
+
+### Enhancement
+
+* in_tail: add option to skip long lines (max_line_size)
+  https://github.com/fluent/fluentd/pull/3565
+
+### Bug fix
+
+* out_file: Make append option work without compression
```

Comment:
This bug reproduces only on macOS, I want to add a note for it.
See also `Releaes Note` in https://github.com/fluent/fluentd/pull/3579#issue-1085787780

Reply:
d1f33da added the affected macOS and Ruby versions.